packages/appium/lib/extension-config.js

```diff
@@ -203,7 +203,13 @@ export default class ExtensionConfig {
 
   require (extName) {
     const {mainClass} = this.installedExtensions[extName];
-    return require(this.getExtensionRequirePath(extName))[mainClass];
+    const reqPath = this.getExtensionRequirePath(extName);
+    const reqResolved = require.resolve(reqPath);
+    if (process.env.APPIUM_RELOAD_EXTENSIONS && require.cache[reqResolved]) {
```

Comment:
how about `APPIUM_FORCE_EXTENSIONS_REALOAD`?

Reply:
hmm, that makes it longer, and it will never reload them under any circumstances otherwise, so I don't see the need for 'FORCE' here